docs/v0.2.0/conf.py

```diff
@@ -20,7 +20,7 @@
 
 # General information about the project.
 version = os.environ["BUILD_VERSION"]
-envoyVersion = os.environ["ENVOY_VERSION"]
```

Comment:
why build doc for a released version?

Reply:
Changes could be backport to a released version, it is common for building versioned docs.